fathom/src/source.rs

```diff
@@ -172,6 +183,13 @@ pub struct ByteRange {
     end: BytePos,
 }
 
+#[test]
+
```

Comment:
Nit: space between the attribute and the definition (not sure if there’s a rustfmt option for this?)